blocks/blog-socials/blog-socials.js

```diff
@@ -0,0 +1,34 @@
+export default function decorate(block) {
+  const topics = document.createElement('div');
+  const tags = document.querySelectorAll('head meta[property="article:tag"]');
+  topics.append('Topics: ');
+  const tagLinks = [];
+  tags.forEach((tag) => tagLinks.push(`<a href="/blog/tag?tag=${encodeURIComponent(tag.content)}">${tag.content}</a>`));
```

Comment:
did you consider using `.map` for this instead?

Reply:
Actually I did, but tags is not an array type so cannot use map.